x-pack/legacy/plugins/siem/cypress/README.md

```diff
@@ -156,7 +156,7 @@ In that case in the hooks of the test use the function `esArchiverLoad` to load
 Example:
 
 ```typescript
-import { esArchiverLoad, esArchiverUnload } from '../tasks/es_archiver';
+import { esArchiverLoad, esArchiverUnload } from '../tasks/archiver';
```

Comment:
I think it's okay for this to stay as it was, as it's coupled to the `es_archiver` script that we do not control.

Reply:
++ @rylnd on keeping these as `es_archiver as it maintains consistent terminology with the dependency and helps with discoverable as well. Also, in that other [other thread](https://github.com/elastic/kibana/pull/62747#discussion_r405130977) I too was referring to the terminology in the README ("archives" instead of "sets of data").